pkg/policies/flowcontrol/resources/classifier/classification-engine.go

```diff
@@ -111,9 +103,18 @@ func (c *ClassificationEngine) populateFlowLabels(ctx context.Context,
 
 	mmResult := mm.Match(labelsForMatching)
 
-	previews := mmResult.previews
-	for _, preview := range previews {
-		preview.AddHTTPRequestPreview(inputValue)
+	// Extract interface{} from ast.Value
+	ifaceRequest, err := ast.ValueToInterface(input, valueResolver{})
+	if err != nil {
+		log.Errorf("failed to convert value to interface: %v", err)
```

Comment:
Nit: `log.Bug()` would be more appropriate perhaps? (So that we're not spammed by logs if this happens on man requests)